libvcx/src/api/wallet.rs

```diff
@@ -891,6 +891,9 @@ pub extern fn vcx_wallet_export(command_handle: CommandHandle,
 /// according to fields provided in import_config
 /// Cannot be used if wallet is already opened (Especially if vcx_init has already been used).
 ///
+/// Note this only works for default storage type (file), as currently this function does not let
+/// you pass down information about wallet storage_type, storage_config, storage_credentials.
+///
```

Comment:
Is `vcx_wallet_import` somehow related to the rest of this PR?

Reply:
Not really, it was just something I've found out in the process, so I've added this information to docs.